src/rollup-plugins/rollup-plugin-remote-cdn.ts

```diff
@@ -59,7 +59,9 @@ export function rollupPluginDependencyCache({
         const pinnedUrlPath = response.headers['x-pinned-url'] as string;
         const typesUrl = typesUrlPath && `${PIKA_CDN}${typesUrlPath}`;
         const pinnedUrl = pinnedUrlPath && `${PIKA_CDN}${pinnedUrlPath}`;
-        await cacache.put(RESOURCE_CACHE, cacheKey, response.body, {
+        // @ts-ignore - text is missing from type definition
```

Comment:
:(

Reply:
Filed upstream https://github.com/DefinitelyTyped/DefinitelyTyped/issues/46135

Will try to get around to submitting a PR for it

Reply:
https://github.com/DefinitelyTyped/DefinitelyTyped/issues/46135